api/.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+# This run in the mattermost gitlab internal instance
```

Comment:
We can remove this 🚀 in favour of https://github.com/mattermost/delivery-platform/pull/5